BookPlayer/Models/Playlist+CoreDataClass.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Playlist+CoreDataClass.swift
+//  BookPlayer
+//
+//  Created by Gianni Carlo on 5/9/18.
+//  Copyright © 2018 Tortuga Power. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+public class Playlist: LibraryItem {
+    convenience init(title: String, books: [Book], context: NSManagedObjectContext) {
+        let entity = NSEntityDescription.entity(forEntityName: "Playlist", in: context)!
+        self.init(entity: entity, insertInto: context)
+        self.identifier = title
```

Comment:
I didn't see this comment. that init is only called when the book is created for the first time, after its content is processed and the hash generated, the identifier is updated to reflect the hash. Mmmm this could be reworked, and instead of creating the book and comparing with the existing ones, I could just compare the hash that I'll be getting anyway 🤔